vortex-serde/src/layouts/reader/builder.rs

```diff
@@ -99,7 +99,7 @@ impl<R: VortexReadAt> VortexLayoutReaderBuilder<R> {
         len as usize
     }
 
-    async fn read_footer(&mut self) -> VortexResult<Footer> {
+    pub async fn read_footer(&mut self) -> VortexResult<Footer> {
```

Comment:
let's not make it pub for now

Reply:
I need this for the benchmarks, I think in general its useful as a user to be able to read the file's schema.